### ISSUE TITLE
Type the remainder of src/util

### ISSUE THREAD

**Issue by tihuan** [chanzuckerberg/cellxgene#2368]
Type the remainder of src/util

**Comment by tihuan** [comment]
One file blocked by Annomatrix PR - centroid , but we'll send out a PR for the rest first!

**Comment by tihuan** [comment]
Created a new ticket for typing Centroid [here](https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/corpora-data-portal/1359)

**Comment by tihuan** [comment]
Closing this one, since it's now merged into main! Thanks so much again, Seve and Colin!
